### DISCUSSION TITLE
draw/utils/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#1203] [Docs]
# draw/utils/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/draw/utils/

**Comment by Jaydeep077** [comment]
module 'supervision' has no attribute 'calculate_dynamic_line_thickness'

sv.__version__
'0.20.0'

**Reply by LinasKo** [reply]
You're right, thanks!

It's been renamed to `calculate_optimal_line_thickness`.

Hey @SkalskiP, docs already have `optimal_line` changes but haven't been deployed.
Another entry is not entioned. Can you help me merge [PR fixing that](https://github.com/roboflow/supervision/pull/1204) and then build / deploy them? I'm not sure about the process.

**Reply by Jaydeep077** [reply]
Thanks 👍

**Comment by AsadShibli** [comment]
AttributeError: module 'supervision' has no attribute 'calculate_dynamic_font_scale'

**Reply by LinasKo** [reply]
I believe the current version is `calculate_dynamic_text_scale`, with it being changed to `calculate_optimal_text_scale` in the next release.

What were you running that prompted this error?

**Reply by SkalskiP** [reply]
@AsadShibli, I'm sorry for the lack of naming continuity :/ I'm also really curious what were you running that prompted this error?

**Comment by NakacwaOlivia** [comment]
module 'supervision' has no attribute 'ByteTrack'

Spervision version 0.8.0

Kindly help out

**Reply by LinasKo** [reply]
Please upgrade to a later version.

`pip install supervision` or `pip install supervision==0.21.0`.

**Reply by NakacwaOlivia** [reply]
Thank you @LinasKo
